.github/workflows/build.yml

```diff
@@ -1,101 +0,0 @@
-name: Build and Test
-on:
-  pull_request:
```

Comment:
This was the exact same as the other workflow file, just with different events. I combined the events and deleted this file

Reply:
And then I remembered that we did this for a reason